src/index.html

```diff
@@ -90,17 +90,18 @@
         limitFrames4D: 3,
         displaySliceInfo: true,
         displaySliceScale: 0.4,
+        enableBorderHighlight: true,
         onLocationChange: handleLocationChange,
       };
       var nv1 = new Niivue(defaults);
       nv1.setRadiologicalConvention(false);
       nv1.attachTo("gl1");
       nv1.setSliceType(nv1.sliceTypeMultiplanar);
       nv1.opts.dragMode = nv1.dragModes.pan;
-      nv1.graph.autoSizeMultiplanar = true;
```

Comment:
Why were those settings commented out?

Reply:
Just for testing purpose. I commented out the graph view. Will be back in with the next push